src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/readytolist/ReadyToListAboutToSubmitHandlerTest.java

```diff
@@ -73,13 +81,37 @@ public void returnAnErrorIfCreatedInGapsFromIsAtValidAppeal() {
 
     @Test
     public void returnAnErrorIfCreatedInGapsFromIsNull() {
-        sscsCaseData = sscsCaseData.toBuilder().createdInGapsFrom(null).build();
+        buildRegionalProcessingCentreMap(HearingRoute.GAPS);
+        sscsCaseData = sscsCaseData.toBuilder().region("TEST").createdInGapsFrom(null).build();
         when(caseDetails.getCaseData()).thenReturn(sscsCaseData);
+
+
         PreSubmitCallbackResponse<SscsCaseData> response = handler.handle(ABOUT_TO_SUBMIT, callback, USER_AUTHORISATION);
 
         assertEquals("Case already created in GAPS at valid appeal.", response.getErrors().toArray()[0]);
     }
 
+    @Test
+    public void givenHearingCreatedAndListAssistIfTypeListAssist() {
```

Comment:
The method name doesn't make sense

Reply:
Replaced name to "givenHearingCreated_withListAssist_thenCheckIfHearingType_isListAssist"